src/vault/client/http.clj

```diff
@@ -102,7 +104,7 @@
       (:http-opts client)
       {:accept :json
        :as :json}
-      req
+      (dissoc req :headers)
```

Comment:
What does this accomplish? Doesn't the `merge` override this field anyway?

Reply:
this is removing the unneeded `:headers` entry from the request body itself.

Reply:
ahhh! for some reason I kept scanning that as separate arguments to `do-api-request`.